src/components/pages/About.vue

```diff
@@ -13,11 +21,41 @@ export default {
 
   props: {
     state: Object
-  }
+  },
+
+  data: () => ({
```

Comment:
You shouldn't use data as a source of static text.

Put the text directly in the template, please.